Aquifer/Simple.swift

```diff
@@ -62,23 +62,24 @@ public func for_<UO, UI, DI, DO, NI, NO, FR>(p: Proxy<UO, UI, DI, DO, FR>, _ f:
     return p |>> f
 }
 
-infix operator <~ {
-associativity left
-precedence 130
-}
 
-/// Composes two loops to yield one large loop.
-public func <~ <IS, UO, UI, DI, DO, NI, NO, FR>(f: IS -> Proxy<UO, UI, DI, DO, FR>, g: DO -> Proxy<UO, UI, NI, NO, DI>) -> IS -> Proxy<UO, UI, NI, NO, FR> {
+//infix operator ~> {
+//associativity right
+//precedence 130
+//}
```

Comment:
Need to mark this with a comment describing why it was removed - at least until #13 is solved